tile/targets/cpu/jit.cc

```diff
@@ -216,7 +219,6 @@ Compiler::Compiler(llvm::LLVMContext* context, const Config& config)
 }
 
 ProgramModule Compiler::CompileProgram(const stripe::Block& program) {
-  IVLOG(4, program);
```

Comment:
seems harmless, easy to restore later if useful

Reply:
Added it back in.